backend/chainlit/data/storage_clients/azure.py

```diff
@@ -79,3 +80,50 @@ async def upload_file(
         except Exception as e:
             logger.warn(f"AzureStorageClient, upload_file error: {e}")
             return {}
+
+
+class AzureBlobStorageClient(BaseStorageClient):
```

Comment:
Creating a separate Azure client for blobs as it seems more appropriate to cloud storage for attachments.